jobs/scheduler/templates/application.properties.erb

```diff
@@ -38,6 +38,9 @@ client.ssl.trust-store=/var/vcap/data/certs/scalingengine/cacerts
 client.ssl.trust-store-password=123456
 client.ssl.protocol=TLSv1.2
 
+#Quartz
+org.quartz.scheduler.instanceName=app-autoscaler
```

Comment:
If we have multiple instances of the scheduler, does this name need to be unique? We may just want to use the bosh name which will have the name and index. It would also make it easier to debug if you look in the DB and you can identify the BOSH VM that it was reserved by.